orbit/pkg/update/runner.go

```diff
@@ -28,6 +29,20 @@ type Runner struct {
 	opt         RunnerOptions
 	cancel      chan struct{}
 	localHashes map[string][]byte
+	mu          sync.Mutex
+}
+
+// UpdateRunnerOptTargets updates the RunnerOptions.Targets with the given target
+func (r *Runner) UpdateRunnerOptTargets(target string) {
+	// check if target already exists
+	for _, t := range r.opt.Targets {
+		if t == target {
+			return
+		}
+	}
+	r.mu.Lock()
```

Comment:
Must move to the top of the method. `r.opt.Targets`, read access must be protected too.